amd/src/learningmap.js

```diff
@@ -312,15 +317,57 @@ export const init = () => {
          * @param {*} evt
          */
         function startDrag(evt) {
-            evt.preventDefault();
+            if (evt.cancelable) {
+                evt.preventDefault();
+            }
             if (evt.target.classList.contains('learningmap-draggable')) {
+                if (evt.type == 'touchstart') {
```

Comment:
It would reduce complexity and improve readability if the touch events could be handled by separate event listener functions (could be added in the block at line 222 ff). These functions could do their touch stuff and after that call the "normal" event handler.

Reply:
Done